src/App.js

```diff
@@ -29,6 +38,13 @@ function App() {
   return (
     <div className="App">
       <Button onClick={getPokemones} variant="outlined">get data</Button>
```

Comment:
Create a `div` that contains inside your Button and Autocomplete components. You're gonna apply some css it has a property call `style` inside.   `<div style={{display:'flex'}}>` it aligns your components. Read about display property in css.